pymatgen/analysis/magnetism/analyzer.py

```diff
@@ -481,10 +481,15 @@ def number_of_unique_magnetic_sites(self, symprec: float = 1e-3, angle_tolerance
     @property
     def ordering(self) -> Ordering:
         """Applies heuristics to return a magnetic ordering for a collinear
-        magnetic structure. Result is not guaranteed for correctness.
+        magnetic structure. Result is not guaranteed to be correct, just a best
+        guess. Tolerance for minimum total magnetization to be considered
+        ferro/ferrimagnetic is 1e-8.
```

Comment:
I am wondering if we should make this adaptable when defining the instance the class if we are not sure about the tolerance yet.

Reply:
ah, wanted to add that but forgot. PRs welcome!

Reply:
@kaueltzen will hopefully open another PR as it was one of the questions in the PR!